ETL/Pipeline/extract.py

```diff
@@ -0,0 +1,61 @@
+"Extract Script: Pulls current price and sale status from ASOS API"
+from datetime import datetime
+import requests
+from lambda_multiprocessing import Pool
```

Comment:
```suggestion
"Extract Script: Pulls current price and sale status from ASOS API"

from datetime import datetime

import requests
from lambda_multiprocessing import Pool
```
Adding spacing looks better and separates the python library from third party libraries

Reply:
This is neat. I've made the changes in my most recent push.